source/MRMesh/MRSceneSettings.h

```diff
@@ -38,6 +38,8 @@ class SceneSettings
         FeaturePointSize,
         // Size of point subfeatures (various centers).
         FeatureSubPointSize,
+        // 
+        FeatureAmbientCoef,
```

Comment:
No need in Feature prefix, also pls add comment, better name SelectionAmbientModifier